src/Exporter/DataItemByExpElementMatchFinder.php

```diff
@@ -109,7 +109,6 @@ private function tryToMatchDataItemForWikiNamespaceUri( $uri ) {
 	private function tryToMatchNamespaceName( $name ) {
 		// try the by far most common cases directly before using Title
 		$namespaceName = str_replace( '_', ' ', $name );
-		$namespaceId = -1;
 
```

Comment:
The next line reassigns the value. So given that the initial assign did not cause side effects, it is dead code. IDE agrees